crdb/tx.go

```diff
@@ -21,6 +21,7 @@ package crdb
 import (
 	"database/sql"
 
+	"github.com/jinzhu/gorm"
```

Comment:
The addition of the dependency on gorm here is unfortunate and a bit undesirable. I wonder if there is a way to support gorm without adding such a dependency.